packages/fxa-payments-server/src/routes/Subscriptions/SubscriptionItem.tsx

```diff
@@ -25,12 +25,14 @@ type SubscriptionItemProps = {
   updatePayment: Function;
   cancelSubscriptionMounted: Function;
   cancelSubscriptionEngaged: Function;
+  cancelSubscriptionStatus: any;
```

Comment:
Instead of `any`, I think you should be able to use [a more specific `CancelSubscriptionFetchState` type](https://github.com/mozilla/fxa/blob/eaa4fa4cdd33b1435c9c7ede4afc654ff1e836ac/packages/fxa-payments-server/src/routes/Subscriptions/index.tsx#L75)

Reply:
Thanks, meant to make a comment asking about that in my pr. Glad you caught it :)